tools/maxquant/maxquant_wrapper.py

```diff
@@ -589,63 +587,61 @@ def setup_database(options):
     database_name = database_name.replace(" ", "_")
     (database_basename, extension) = os.path.splitext(database_name)
     database_destination = get_unique_path(database_basename, ".fasta")
+    database_id = options.andromeda_config
+    if(database_id == 'all_include'):
+        database_regex = '(.*)'
+    elif(database_id == 'all_after'):
+        database_regex = '>(.*)'
+    elif(database_id == 'upTofirst_space'):
+        database_regex = '>([^ ]*)'
+    elif(database_id == 'ipi_acc'):
+        database_regex = '>IPI:([^\| .]*)'
+    elif(database_id == 'ncbi_acc'):
+        database_regex = '>(gi\|[0-9]*)'
+    elif(database_id == 'upTofirst_tab'):
+        database_regex = '>([^\t]*)'
+    elif(database_id == 'uniprot'):
+        database_regex = '>.*\|(.*)\|'
+    elif(database_id == 'upTofirst_slash'):
+        database_regex = '>([^\|]*)'
+
+
     assert database_destination == os.path.basename(database_destination)
     symlink(database_path, database_destination)
 
     database_conf = get_env_property("MAXQUANT_DATABASE_CONF", None)
     if not database_conf:
-        exe_path = which("MaxQuantCmd.exe")
+        exe_path = "C:\MaxQuant\\bin\MaxQuantCmd.exe"
```

Comment:
This is not ideal - you should make sure ``MaxQuantCmd.exe`` is on LWR/Pulsar's path instead of changing the tool. If this is the most common place MaxQuant is installed to though - it may make sense to fallback to this default if which cannot determine which MaxQuant to use.